pkg/resource/deploy/step_generator.go

```diff
@@ -425,10 +425,18 @@ func (sg *stepGenerator) GenerateDeletes() []Step {
 				// Regardless, it is better to admit strange behavior in corner cases than it is to crash the CLI
 				// whenever we see multiple deletes for the same URN.
 				// contract.Assert(!sg.deletes[res.URN])
+				if sg.pendingDeletes[res] {
+					logging.V(7).Infof(
+						"Planner ignoring pending-delete urn '%v' that was already deleted", res.URN)
```

Comment:
Can we change the log to `...pending-delete resource (%v, %v)...", res.URN. res.ID`?

Reply:
will do.